controllers/bootstrap/init.go

```diff
@@ -111,6 +111,17 @@ func (b *Bootstrap) InitResources(manualManagement bool) error {
 		return err
 	}
 
+	// Check Saas or Multi intances Deployment
+	controlNs := util.GetControlNs(b.Reader)
+	if len(controlNs) > 0 {
+		if err := b.CreateNamespace(controlNs); err != nil {
+			klog.Errorf("Failed to create control namespace: %v", err)
+			return err
+		}
+	} else {
+		controlNs = b.MasterNamespace
+	}
+
```

Comment:
Just a thought, could we add `ControlNamespace` and `SaaSenabled` into the `Bootstrap` structure, like what we did for MasterNamespace. Then we don't need to trigger the function everytime